src/_guides/language/sound-dart.md

```diff
@@ -37,41 +37,47 @@ developer experience.
 Strong mode Dart adds only a few additional rules beyond that for classic
 Dart&mdash;mostly you clarify code where the types are ambiguous or
 incorrect. In fact, most strong mode errors can be fixed by adding type
-annotations to your Lists and Maps. The following code shows valid
-but unsound Dart code. The `fn()` function prints an integer list.
-The `main()` function creates a list of integers and passes it to `fn()`:
+annotations to your Lists and Maps.
+For example, in the following code the `fn()` function prints an integer list,
```

Comment:
Let's split this into two paragraphs, so we don't bury the tip.

We might even boldface **adding type annotations to your Lists and Maps.** What do you think?

Reply:
Paragraphs split.

> We might even boldface **adding type annotations to your Lists and Maps.** What do you think?

My first reaction would be to not put it in bold. I have a proposal to rework the initial paragraphs. Maybe we can revisit using bold or not then